tests/testthat/test-read.R

```diff
@@ -74,6 +75,61 @@ test_that("Reading H5AD works with native reader", {
     expect_identical(colnames(colData(sce)), "cell_type")
 })
 
+test_that("Reading v0.8 H5AD works with native reader", {
+    sce_py <- readH5AD(file_v08)
+    sce_r <- readH5AD(file_v08, reader='R')
+
+    expect_identical(rownames(sce_py), rownames(sce_r))
+    expect_identical(colnames(sce_py), colnames(sce_r))
+
+    # rowData is just an empty DataFrame here
+    expect_identical(rowData(sce_py), rowData(sce_r))
+
+    # colData colnames not in same order, but check the sets are the same
+    expect_true(setequal(colnames(colData(sce_py)), colnames(colData(sce_r))))
+
+    # check colData columns that Python reader is able to handle
+    good_coldat_columns <- c('cell_type', 'dummy_bool', 'dummy_int',
+                             'dummy_num', 'dummy_num2')
```

Comment:
Is it the `NA` thing that doesn't work with the Python reader or something else?